### PR TITLE
3366 - whats new in 0.8

### LINKED ISSUE #3366: Add `What's new in v0.8` page
**Is your feature request related to a problem? Please describe.**
Need to prepare the `What's new in v0.8` page for v0.8 release.

### PR DESCRIPTION
**Pull request by wyli**
Signed-off-by: Wenqi Li <wenqil@nvidia.com>

part of #3351
closes #3366

### Description
adds a what's new page

### Status
**Ready/Work in progress/Hold**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: docs/source/whatsnew_0_8.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# What's new in 0.8 🎉🎉
+
+- Differentiable neural network topology search
+- Multiple instance learning for digital pathology WSI analysis
+- Self-supervised representation learning modules
```
Comment (finalelement) [inline docs/source/whatsnew_0_8.md:5]: let's just remove 'modules'?

File: docs/source/whatsnew_0_8.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+MONAI starts to explore self-supervised representation learning in this
+milestone release. The Vision Transformer has been extended to learn from self-supervised
+reconstruction tasks with various data augmentation and a regularized
+contrastive loss. The weights of the pre-trained backbone could be used to
+enhance the performance of the novel downstream deep learning tasks.
+
+The [tutorial](https://github.com/Project-MONAI/tutorials/tree/master/self_supervised_pretraining)
+shows how to generate a good set of pre-trained weights using unlabeled data
+with self-supervised tasks, then use the pre-trained weights to perform
+fine-tuning on a fully supervised volumetric segmentation task.
```
Comment (finalelement) [inline docs/source/whatsnew_0_8.md:41]: before the sentence ends, we can add 'using UNETR which is a transformer based model'

File: docs/source/whatsnew_0_8.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+MONAI integrates `DiNTS`: [Differentiable Neural Network Topology Search for 3D
+Medical Image Segmentation](https://arxiv.org/abs/2103.15954). The neural
+architecture search module supports flexible multi-path topology search with
+high search efficiency and budgeted memory usage.
+
+It provides a topology guaranteed discretization algorithm and a
+discretization-aware topology loss for the search stage to minimize the
+discretization gap. The module is memory usage aware and is able to search 3D
+networks with different GPU memory requirements. For more details, please checkout the
+[DiNTS tutorial](https://github.com/Project-MONAI/tutorials/tree/master/neural_architecture_search).
```
Comment (finalelement) [inline docs/source/whatsnew_0_8.md:18]: This link is broken, leads me to 404.

**Review by Nic-Ma — APPROVED** [review]

**Pull request merged**